tour-tip-group/config.json

```diff
@@ -20,8 +21,10 @@
     "content": {
       "title": "Over to you",
       "body": "Now you’ve seen what you can do, make the most of the new FT.com.",
-      "ctaUrl": "/",
-      "ctaLabel": "Get started"
+      "ctas": [{
+        "ctaUrl": "/",
```

Comment:
Low priority request: Could we change `ctaUrl`/`ctaLabel` to `url`/`label` por favor?